tests/test_normalized_gym.py

```diff
@@ -0,0 +1,30 @@
+import gym
+from rllab.envs.normalized_gym_env import NormalizedGymEnv
```

Comment:
PEP8: import grouping

Reply:
Yes, I agree about the utils. I asked @jonashen to add that into his pr since he is doing the refactoring of gym.Env so let me reopen this after his pr.